### PR TITLE
[FIX] in-place damping bug in `KFACInverseLinearOperator`

### PR DESCRIPTION
**Pull request by runame**
When `use_exact_damping=True` and `KFACLinearOperator._separate_weight_and_bias=True`, the damping will be repeatedly added to the eigenvalues corresponding to the bias parameters with every matrix-vector product.

### CODE REVIEW

**Comment by f-dangel** [comment]
Great catch! Can we merge this after #149? This PR refactors the `KFAC` class a lot, and I'm afraid it might lead to having to resolve a lot of nasty merge conflicts.

**Comment by runame** [comment]
Sure, but this PR only changes a single underscore in `inverse.py`, so should not result in any significant merge conflict. The other changed lines are in the test file.

**Review by f-dangel — APPROVED** [review]
LGTM. Could you update the changelog before merging?

**Pull request merged**
